### ISSUE TITLE
Error: Error parsing scripts.zs:10 -- ; expected

### ISSUE THREAD

**Issue by BlockBite** [stanhebben/MineTweaker3#462]
trying to delete any craft and MT send me message with this error. 
https://pastebin.com/ihmvs46b

**Comment by VariableVixen** [comment]
A semicolon was expected and not found. On line four, you are missing a semicolon at the end of the line. These two things are very closely related.

**Comment by BlockBite** [comment]
now scripts is reloading, but I don't see changes

**Comment by VariableVixen** [comment]
Reloading has always been a bit finicky, and IIRC it was described - by the modder - as a bit of a hack. Also... I looked at your code again. There doesn't seem to be any reason to import NEI. You're using the base mod's vanilla `recipes` object. My advice: if there are no errors, remove the `import mods.nei.NEI` line and restart the game instead of reloading the recipes. Sometimes a mod tries to cache recipes itself and that breaks reloading.

Finally, these are FSP's recipes. I know IC2 uses a custom crafting manager for its recipes, not the vanilla one. Is FSP compatible with MT3, or does it do something like that?

**Comment by BlockBite** [comment]
No, still basic crafts. `Finally, these are FSP's recipes. I know IC2 uses a custom crafting manager for its recipes, not the vanilla one. Is FSP compatible with MT3, or does it do something like that?` I'm using recipes in a regular workbench. Don't think, about I need another addon for FSP

**Comment by BlockBite** [comment]
Also, FSP using that
![default](https://user-images.githubusercontent.com/42101002/44623806-ac805680-a8e1-11e8-8456-9c5723b1628a.png)

**Comment by BlockBite** [comment]
soo... code looks rn https://pastebin.com/ud564krd

**Comment by BlockBite** [comment]
ok, I have deleted all scripts, left only 1 line of craft - which deletes craft in botania "recipes.remove(<Botania:lexicon>);", but it's don't working :(

**Comment by Preta-Crowz** [comment]
`recipes.remove(<Botania:lexicon>);`
correct syntax is `<mod:id:meta>`

**Comment by VariableVixen** [comment]
`meta` is optional, and if omitted is considered to be `0`.
